WordPress/Classes/ViewRelated/Stats/Today Widgets/WidgetStyles.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import NotificationCenter
+
+class WidgetStyles: NSObject {
+
+    static let primaryTextColor: UIColor = .text
+    static let secondaryTextColor: UIColor = .textSubtle
+
+    private static var separatorColor: UIColor = {
+        if #available(iOS 13, *) {
+            return UIColor(white: 1.0, alpha: 0.5)
```

Comment:
Does `.separator` not work for iOS 13?

Reply:
Why yes, yes it does. Thank you sir. Tis been changed.